.ci/docker/common/install_cudnn.sh

```diff
@@ -4,6 +4,8 @@ if [[ -n "${CUDNN_VERSION}" ]]; then
     # cuDNN license: https://developer.nvidia.com/cudnn/license_agreement
     mkdir tmp_cudnn
     pushd tmp_cudnn
+    if [[ ${CUDA_VERSION:0:4} == "12.6" ]]; then
+        CUDNN_NAME="cudnn-linux-x86_64-9.5.1.17_cuda12-archive"
     if [[ ${CUDA_VERSION:0:2} == "12" ]]; then
```

Comment:
if [[ ${CUDA_VERSION:0:2} == "12" ]]; then ...

it seems to me that CUDA 12.6 will evaluate to true for this one ? 

Perhaps, before that, is this missing a fi?  

Perhaps we need to change the if to elif?  

cc @Skylion007